actions/README.md

```diff
@@ -1,49 +1,77 @@
 
+Decorate your Python function with the `@action` decorator:
 
-```python
+```py
 from robocorp.actions import action
 
 @action
 def sum_numbers(a: float, b: float) -> float:
     ...
 ```
 
-Note: it's expected that the action arguments and action return are properly
-typed (if a type is not specified, `str` is considered for it).
+**And your function is now an ⚡️ action!**
 
-Note: only `int`, `float`, `str` and `bool` types are currently supported.
+You can now run and debug your action by Starting Action Server with `action-server start` and accessing the UI at http://localhost:8080.
 
-The entry points for the values should be passed by the `Robocorp Action Server`,
-but it's also possible to provide them in the command line by passing them in
-named arguments after a `--`.
+> [!NOTE]
+> Action inputs and outputs support only `int`, `float`, `str`, and `bool` types.
 
-Example:
+## Describe your action
 
+For an action's purpose and usage to be understood by AI models _(and humans)_ it needs to be documented correctly.
 
-```sh
-python -m robocorp.actions -- --a=2 --b=3
+To do that, use [Google Style Docstring](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings) to write a clear and concise description on what your action does and document the action inputs and expected output:
+
+```py
+@action
+def get_weather_forecast(city: str, days: int, scale: str = "celsius") -> str:
+    """
+    Returns weather conditions forecast for a given city.
+
+    Args:
+        city (str): Target city to get the weather conditions for
+        days: How many day forecast to return
+        scale (str): Temperature scale to use, should be one of "celsius" or "fahrenheit"
+
+    Returns:
+        str: The requested weather conditions forecast
+    """
+    ...
+```
+
+> [!TIP]
+> Experiment with and iterate the exact documentation wording to get more predictable results when using your action with AI apps.
+
+---
+
+### Consequential flag
+
+You can explicitly provide the `is_consequential` flag for an action to mark it's operations as "must always prompt the user for confirmation before running" by [OpenAI GPTs](https://platform.openai.com/docs/actions) _(and possibly by other providers)_. If set to `False`, the user will be provided with an "always allow" feature.
+
+```py
+@action(is_consequential=True)
+def get_weather_forecast(city: str, days: int, scale: str = "celsius") -> str:
 ```
 
-When used in the `Robocorp Action Server`, an Open API spec (`openapi.json`) 
-is provided to be comformat to these parameters (the `Robocorp Action Server` 
-manages passing the arguments accordingly and returning the result afterwards).
+### Execution
+
+To get the full benefits of your actions, the suggested way to run them is using Action Server. But it's also possible to do that directly in command line by passing the named arguments:
+
+```sh
+python -m robocorp.actions -- --city=Helsinki --days=3
```

Comment:
The command does not work.
I think it needs to be: python -m robocorp.actions run -- --city=Helsinki --days=3

Reply:
Updated